subsys/debug/gpio_frontend/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if LOG_FRONTEND
+config LOG_FRONTEND_GPIO
+	bool "Enable GPIO frontend"
+	help
+	  When enabled frontend is using GPIO pins to output logs.
+
+config LOG_FRONTEND_PORT_NUMBER
+	int "Choose port number for GPIO frontend"
+	depends on LOG_FRONTEND
```

Comment:
that should rather depend on `LOG_FRONTEND_GPIO`. Actually whole section should be surrounded by if endif and then those `depends on` can be removed.